node/src/components/contract_runtime.rs

```diff
@@ -483,13 +484,8 @@ where
                         responder,
                     } => {
                         trace!(?trie_key, "read_trie request");
-                        let engine_state = Arc::clone(&self.engine_state);
-                        let metrics = Arc::clone(&self.metrics);
+                        let result = self.read_trie(trie_key);
                         async move {
-                            let correlation_id = CorrelationId::new();
-                            let start = Instant::now();
-                            let result = engine_state.read_trie(correlation_id, trie_key);
-                            metrics.read_trie.observe(start.elapsed().as_secs_f64());
```

Comment:
@dwerner Got rid of `read_trie` metric; it is unused in production.

Reply:
Ah, you are right... moving forward these metrics are a maintenance burden, especially if they are not used.